lfucg-exactions/server/plats/serializers.py

```diff
@@ -171,7 +171,13 @@ def to_internal_value(self, data):
             return None
 
     def to_representation(self, obj):
-        return PlatSerializer(obj).data
+        return {
+            'id': obj.id,
+            'name': obj.name,
+            'slide': obj.slide,
+            'buildable_lots': obj.buildable_lots,
+            'non_buildable_lots': obj.non_buildable_lots,
+        }
```

Comment:
https://stackoverflow.com/questions/5352546/extract-subset-of-key-value-pairs-from-python-dictionary-object